.claude/agent-memory/archgate-developer/project_adr_corpus_maintenance.md

```
@@ -0,0 +1,12 @@
+---
+name: project-adr-corpus-maintenance
+description: How to trim or restructure the ADR corpus without losing governance — briefing budget, the adr-author floors, and why fabricating content is the wrong fix
+metadata:
+  type: project
+---
+
+- **The two briefed sections get the same cap applied independently, so rebalancing beats cutting.** `review-context --verbose` truncates `Decision` and `Do's and Don'ts` at `DEFAULT_MAX_SECTION_CHARS` each — not a pooled allowance across the two. Because the caps are independent, moving a normative rule out of an over-cap section into one with headroom increases total visible governance even though nothing was deleted. `archgate check` reports every overflow as a built-in `[briefing]` advisory; it is non-blocking because some ADRs cannot shrink without losing a normative clause. ARCH-005 went from ~17% of its Do's visible to Decision-plus-all-ten-Do's visible this way. This is legitimate; what GEN-005 forbids is shuffling prose between them purely to clear a warning without increasing what a consumer actually sees.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)